web/forms.py

```
@@ -527,6 +527,50 @@
+            'text': TailwindTextarea(attrs={'rows': 3, 'placeholder': 'Question text'}),
+            'question_type': TailwindSelect(),
+            'explanation': TailwindTextarea(attrs={'rows': 2, 'placeholder': 'Explanation for the correct answer'}),
+            'points': TailwindNumberInput(attrs={'min': '1', 'value': '1'}),
+            'image': TailwindFileInput(attrs={'accept': 'image/*'}),
+        }
+    
+    # Additional fields for specialized question types
+    code_starter = forms.CharField(
+        widget=TailwindTextarea(attrs={'rows': 5, 'placeholder': 'Starter code for coding questions'}),
+        required=False,
+        help_text="Starter code for coding questions"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)